hprotoc/Text/ProtocolBuffers/ProtoCompile/Gen.hs

```diff
@@ -740,6 +828,12 @@ standardImports isEnumMod ext lenses =
             | otherwise = ["+","/"]
        lensTH | lenses = [ImportDecl src (ModuleName "Control.Lens.TH") True False False Nothing Nothing Nothing]
               | otherwise = []
+       mapModule
+            | not isMapEntry = []
```

Comment:
imo listing imports for `isMapEntry` and `[]` for `otherwise` is slightly more readable